chrono-tz/src/directory.rs

```diff
@@ -2,4 +2,9 @@
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
 #![allow(dead_code)]
+
+#[cfg(feature = "filter-by-regex")]
 include!(concat!(env!("OUT_DIR"), "/directory.rs"));
+
+#[cfg(not(feature = "filter-by-regex"))]
+include!("prebuilt_directory.rs");
```

Comment:
Could we instead have `#[path = "prebuilt_directory.rs"]` on the `mod` declaration in `lib.rs` conditional on a `cfg()`, or does that not work? Otherwise, can we use a `mod` declaration here and re-export all symbols? I'd like the prebuilt modules to be visible to Rust-Analyzer et al.

Reply:
I think that'd work (the first suggestion), I'll try it.

Reply:
Unfortunately I can't get this to work, because we need the `OUT_DIR` environment variable but the `path` attribute only accepts string literals.

Reply:
For what it's worth rust-analyzer seems to be able to pierce through the `include!` barrier.

Reply:
@djc I don't quite follow what you mean by that. I've tried your previous suggestion but could not get it to work. Are you suggesting me to rename `src/prebuilt/directory.rs` to `src/prebuild_directory.rs`? Or are you suggesting something else?